packages/suite/src/components/suite/DropZone/index.tsx

```diff
@@ -144,14 +133,14 @@ const Label = styled.div`
 `;
 
 export const DropZone = (props: Props) => {
-    const { getWrapperProps, getInputProps, error } = useDropZone(props);
+    const { getWrapperProps, getInputProps, error, filename } = useDropZone(props);
 
     return (
         <Wrapper {...getWrapperProps()}>
             <StyledInput {...getInputProps()} />
             <Label>
                 <StyledIcon icon="CSV" />
```

Comment:
```suggestion
                <StyledIcon icon="BIN_FILE" />
```
.bin file extension icon, ask Ondra for one

Reply:
Added in [`12ec3f54`](https://github.com/trezor/trezor-suite/commit/12ec3f54e4e3f9c7b994c5d9200ece538f70f6d7).